faunadb/__init__.py

```diff
@@ -1,6 +1,29 @@
+import requests
+
 __title__ = "FaunaDB"
 __version__ = "4.1.0"
 __api_version__ = "4"
 __author__ = "Fauna, Inc"
 __license__ = "MPL 2.0"
 __copyright__ = "2020 Fauna, Inc"
+
+
+response = requests.get('https://pypi.org/pypi/faunadb/json')
+latest_version = response.json().get('info').get('version')
+latest_version_arr = latest_version.split(".")
+current_version = __version__.split(".")
+is_new_version_available = False
+
+for i in range(len(current_version)):
+  if latest_version_arr[i] > current_version[i]:
```

Comment:
I understand that this situation is not really possible, but what if `current_version` is 5.0.0 and `latest_version` is 4.2.0. I think the `is_new_version_available` returns true.